http.d.ts

```diff
@@ -111,14 +111,14 @@ export interface IHttpLink {
 export interface IHttpRequestBody {
   description?: string;
   required?: boolean;
-  content: IHttpContent[];
+  contents: IHttpContent[];
 }
 
 export interface IHttpRequest {
-  path?: IHttpPathParam[];
-  query?: IHttpQueryParam[];
-  headers?: IHttpHeaderParam[];
-  cookie?: IHttpCookieParam[];
+  pathParams?: IHttpPathParam[];
+  queryParams?: IHttpQueryParam[];
+  headerParams?: IHttpHeaderParam[];
+  cookieParams?: IHttpCookieParam[];
```

Comment:
Mixed feelings. It is more descriptive, but we have typed language with IDE support for telling us what's hidden behind fields. But yes, still more descriptive ;)

Reply:
I think it is too descriptive, like @StefanDywersant said the IDE does this for you.

Reply:
Don't have a strong opinion either way, leave it to you folks!